boards/silabs/radio_boards/siwx917_rb4338a/board.cmake

```diff
@@ -1,8 +1,12 @@
 # Copyright (c) 2024 Silicon Laboratories Inc.
 # SPDX-License-Identifier: Apache-2.0
-
+# Set the flash file based on signing is enabled or not.
+set(FLASH_FILE ${PROJECT_BINARY_DIR}/zephyr_signed.bin.rps)
+if(NOT EXISTS ${FLASH_FILE})
+    set(FLASH_FILE ${PROJECT_BINARY_DIR}/zephyr.bin.rps)
```

Comment:
Later in the script we refer to `${KERNEL_BIN_NAME}.rps` or to `${CONFIG_KERNEL_BIN_NAME}_signed.bin.rps`